src/Utilities/Compiler/WellKnownTypeNames.cs

```diff
@@ -4,6 +4,7 @@ namespace Analyzer.Utilities
 {
     internal static class WellKnownTypeNames
     {
+        public const string SystemObject = "System.Object";
```

Comment:
Please revert.

Reply:
Sorry, I don't understand what you mean by revert. Meaning don't add this line? Move it to my analyzer class as a private const?

Reply:
Nevermind my comment. Saw the above comments on how to get the object name so I removed this line.